testsuite/integration-arquillian/tests/base/src/test/java/org/keycloak/testsuite/adapter/servlet/OfflineServletsAdapterTest.java

```diff
@@ -129,8 +129,10 @@ public void testServlet() {
             assertCurrentUrlStartsWith(offlineTokenPage);
 
             // assert successful refresh
-            assertThat(offlineTokenPage.getRefreshToken().getId(), not(refreshTokenId));
-            assertThat(offlineTokenPage.getAccessToken().getId(), not(accessTokenId));
+            refreshToken = offlineTokenPage.getRefreshToken();
+            assertThat(refreshToken, notNullValue());
+            assertThat(refreshToken.getId(), not(refreshTokenId));
+            assertThat(refreshToken.getId(), not(accessTokenId));
```

Comment:
I think the last assert should be for accessToken to match the previous behaviour?

Reply:
Oh, nice catch